docs/pages/hdmi_cec_VirtualComponent_ControlPlaneCommands.md

```diff
@@ -0,0 +1,728 @@
+## Control Plane Commands
```

Comment:
should add an overview, what about this?

## Overview of HDMI-CEC Virtual Component Control Plane Commands

The HDMI-CEC virtual component enables comprehensive testing of HDMI-CEC functionalities through the control plane. Testers can interact with the virtual component using four primary types of commands, each serving a distinct purpose:

1. **Command:** Directly trigger specific HDMI-CEC commands between devices, simulating real-world user interactions.

2. **Event:** Simulate HDMI-CEC events like hotplug (device connection/disconnection) to test how devices react.

3. **Config:** Modify the virtual CEC network configuration, changing device types, connections, and capabilities.

4. **State:** Dynamically add or remove devices during testing and print the current status for debugging.

Each command type is structured as a YAML payload, which is outlined in detail in the following sections. These payloads allow testers to precisely control the virtual HDMI-CEC environment, replicating various scenarios and edge cases.